features/step_definitions/user_management_steps.rb

```diff
@@ -46,7 +46,11 @@
 end
 
 Given(/^I am not an admin user$/) do
-  @current_user = Comfy::Cms::User.create!(admin: false, email: 'test@tester.com', password: 'password')
+  @current_user = Comfy::Cms::User.create!(
```

Comment:
Hi @Tr4pSt3R. The change was simply to allow a test to continue to pass. I feel it's not within the scope of this particular PR to refactor the implementation of the test.

If I switched it to using factory girl the only thing that would want to change would be `Comfy::Cms::User.create!` would become `FactoryGirl.create(:user,`. As it's a feature step definition, I would expect the email address/password to potentially be re-used by other steps.

However, it does raise a good point about reviewing the current codebase and seeing if there are things we can maybe make a pass thru and tidy up wholesale.